Sources/SwerverCore/Request/HttpRequestParser.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public class HttpRequestParser {
+    
+    public init() {}
```

Comment:
Is this `public init` needed if the body is empty?

Reply:
If I remove line 5, the parser class will initiate, even if it's empty.

Reply:
@cmvandrevala Public initializers are often so the tests can access the initializer.

Reply:
Ah ok